lua/conform/formatters/prettierd.lua

```diff
@@ -1,5 +1,55 @@
 local fs = require("conform.fs")
 local util = require("conform.util")
+
+local config_file_names = {
+  -- https://prettier.io/docs/en/configuration.html
+  ".prettierrc",
+  ".prettierrc.json",
+  ".prettierrc.yml",
+  ".prettierrc.yaml",
+  ".prettierrc.json5",
+  ".prettierrc.js",
+  ".prettierrc.cjs",
+  ".prettierrc.mjs",
+  ".prettierrc.toml",
+  "prettier.config.js",
+  "prettier.config.cjs",
+  "prettier.config.mjs",
+}
+
+local function read_json(file)
```

Comment:
Let's add some type annotations
```suggestion
---@param file string
---@return nil|table
local function read_json(file)
```